### PR TITLE
Conda env path update 

### LINKED ISSUE #8: Workaround/Update for setting up Conda env on the container
The current `.env` file requires conda to be installed in the container before a custom env can be used. This can be bypassed by mounting the entire `conda` dir found at the netscratch head node to the container rather than just the env folder.

Suggested fix to `.env.example` file:
```
# change this if you have installed miniconda to another location
HOST_CONDA_ENVS_DIR=/netscratch/$USER/miniconda3/
# change this if you use another Enroot image with a different conda location
CONTAINER_CONDA_ENVS_DIR=/opt/conda/
```

By mounting the entire `conda` folder the initial `source /opt/conda/bin/activate` in `activate_and_execute.sh` can never fail allowing the custom env activated when wanted.

### PR DESCRIPTION
**Pull request by jk-tripathy**
Updates the path as described in #8

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -28,9 +28,9 @@ HOST_WORKDIR=`pwd`
 # ensure that this path exists at the host!
 HOST_CACHEDIR=/netscratch/$USER/.cache_slurm
 # change this if you have installed miniconda to another location
-HOST_CONDA_ENVS_DIR=/netscratch/$USER/miniconda3/envs
+HOST_CONDA_ENVS_DIR=/netscratch/$USER/miniconda3/
 # change this if you use another Enroot image with a different conda location
-CONTAINER_CONDA_ENVS_DIR=/opt/conda/envs
+CONTAINER_CONDA_ENVS_DIR=/opt/conda/
```
Comment (ArneBinder) [inline .env.example]: Why are there slashes at the end of the paths?

**Review by ArneBinder — CHANGES_REQUESTED** [review]
One question, see below.

File: .env.example (hunk 1 above)
Comment (jk-tripathy) [reply .env.example]: functionally there is no difference between having the slash in the end vs not having the slash in the end. I believe having the slash in the end is the general UNIX convention, but i can remove those if you want me to?

Comment (ArneBinder) [reply .env.example]: hmm not so sure about that, I found just that, for instance: https://en.wikipedia.org/wiki/Talk:Path_(computing)#Filepath_Misconceptions

Not 100% sure, but I think it is a bit better to remove them.

Comment (jk-tripathy) [reply .env.example]: got it

**Review by ArneBinder — APPROVED** [review]

**Pull request merged**